README.md

```diff
@@ -4,14 +4,119 @@
 
 An OCI container runtime monitor.
 
-Conmon is a monitoring program and communication tool between a container manager (like [podman](https://podman.io/) or [CRI-O](https://cri-o.io/)) and an OCI runtime (like [runc](https://github.com/opencontainers/runc) or [crun](https://github.com/giuseppe/crun)) for a single container.
+Conmon is a monitoring program and communication tool between a
+container manager (like [podman](https://podman.io/) or
+[CRI-O](https://cri-o.io/)) and an OCI runtime (like
+[runc](https://github.com/opencontainers/runc) or
+[crun](https://github.com/giuseppe/crun)) for a single container.
 
-Upon being launched, it double-forks to daemonize and detach from the parent that launched it. It then launches the runtime as its child.  This allows managing processes to die in the foreground, but still be able to watch over and connect to the child process (the container).
+Upon being launched, it double-forks to daemonize and detach from the
+parent that launched it. It then launches the runtime as its child. This
+allows managing processes to die in the foreground, but still be able to
+watch over and connect to the child process (the container).
 
 While the container runs, conmon does two things:
-* Provides a socket for attaching to the container, holding open the container's standard streams and forwarding them over the socket.
-* Writes the contents of the container's streams to a log file (or to the systemd journal) so they can be read after the container's death.
 
-Finally, upon the containers death, conmon will record its exit time and code to be read by the managing programs.
+  - Provides a socket for attaching to the container, holding open the
+    container's standard streams and forwarding them over the socket.
+  - Writes the contents of the container's streams to a log file (or to
+    the systemd journal) so they can be read after the container's
+    death.
 
-Written in C and designed to have a low memory footprint, conmon is intended to be run by a container managing library. Essentially, conmon is the smallest daemon a container can have.
+Finally, upon the containers death, conmon will record its exit time and
+code to be read by the managing programs.
+
+Written in C and designed to have a low memory footprint, conmon is
+intended to be run by a container managing library. Essentially, conmon
+is the smallest daemon a container can have.
+
+In most cases, conmon should be packaged with your favorite container
+manager. However, if you'd like to try building it from source, follow
+the steps below.
+
+## Depedencies
+
+These dependencies are required for the build:
+
+### Fedora, CentOS, RHEL, and related distributions:
+
+``` shell
+sudo yum install -y \
+  gcc \
+  git \
+  glib2-devel \
+  glibc-devel \
+  make \
+  pkgconfig \
+  runc
```

Comment:
we've listed runc here and a few other places below.  What about crun?  I think Fedora 32+ might not be happy with runc. 

@rhatdan WDYT?

Reply:
Yes crun would be better.

Reply:
BTW crun is NOT always available for most OS: https://pkgs.org/download/crun

How about remove `runc`?

Reply:
I checked both Podman (https://github.com/alvistack/podman.io/blob/master/getting-started/installation.md#build-and-run-dependencies) and CRI-O (https://github.com/cri-o/cri-o/blob/master/tutorials/setup.md#build-and-run-dependencies) also mentioning `runc`.

Due to crun is NOT always available for most OS: https://pkgs.org/download/crun, I would like to suggest just keep it as-is ;-S